src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,8 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    @Override
+    public int compareTo(Person o) {
+        return this.name.toString().compareTo(o.name.toString());
```

Comment:
should you use the `name` attribute directly?  and is there a difference between using `.name` and `.name.toString()`